### PR TITLE
x64 JIT: Remove debug baggage and add peephole opts

### PR DESCRIPTION
**Pull request by wheremyfoodat**

### CODE REVIEW

**Comment by wheremyfoodat** [comment]
Testing GH webhook
image

**Comment by wheremyfoodat** [comment]
Webhook test
image

File: src/jit_no_ir.h [hunk 1]
```diff
@@ -692,9 +692,7 @@ class EmitX64 {
         c.imul(x, y);
         c.mov(dword[REGS + offsetof(JitRegisters, p) + sizeof(u32) * unit], x);
         if (x_sign || y_sign) {
-            c.bt(x, 31);
-            c.setc(x.cvt8());
-            c.and_(x.cvt16(), 0x1);
+            c.shr(x, 31);
```
Comment (wheremyfoodat) [inline src/jit_no_ir.h:695]: Webhook test

**Comment by wheremyfoodat** [comment]
Panda

**Comment by wheremyfoodat** [comment]
Testing webhook
image

**Comment by wheremyfoodat** [comment]
This one should be good to go too, ~~try testing on Linux~~

**Comment by raphaelthegreat** [comment]
Who said test in production :P

**Pull request merged**